.travis.yml

```diff
@@ -0,0 +1,59 @@
+branches:
+  only:
+  - travis_release_build
+  - master
+  - release
+  - "/^v\\d+\\.\\d+(\\.\\d+)?(-\\S*)?$/"
```

Comment:
Travis will assemble release apks on master, release and some tags. The tag format is *v0.1*, *v0.0.1*, *v0.0.1-preview* etc.